### PR TITLE
Add STM32F4xx example

### PR DESCRIPTION
**Pull request by JosefUtbult**
Hello. I'm working on a rust sound card for a university project. I'm gonna try to make it using an STM32F4 processor, and has therefore made an example for that processor. I hope that is of interest for you.

### CODE REVIEW

**Comment by kiffie** [comment]
Of course, this is of interest. I will have a look at it.

Do the ISO endpoints work on this device? I am asking because I tried the STM32F446 a longer time ago. At that time, I observed problems with the ISO endpoint and I needed to patch the USB HAL module. Maybe, I will try your example on my STM32F446 device.

File: examples/stm32f4xx/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[package]
+name = "usb-audio"
+version = "0.1.0"
+authors = ["Kiffie van Haash", "Josef Utbult"]
+license = "MIT"
```
Comment (kiffie) [inline examples/stm32f4xx/Cargo.toml:5]: Could you please replace with ""MIT OR Apache-2.0"?

Comment (JosefUtbult) [reply examples/stm32f4xx/Cargo.toml:5]: Sure! I am not sure to what you are referring to with ISO endpoints, but everything seems to be working as far as I understand

Comment (kiffie) [reply examples/stm32f4xx/Cargo.toml]: Ok. So we have a working stm32 example, which is great.

**Comment by kiffie** [comment]
Thank you very much for this PR.

**Pull request merged**